lib/features/mailbox/presentation/mailbox_controller.dart

```diff
@@ -87,7 +88,11 @@ class MailboxController extends BaseController {
   void onError(error) {}
 
   void _buildTree(List<PresentationMailbox> folderMailboxList) async {
-    folderMailboxTree.value = await _treeBuilder.generateMailboxTree(folderMailboxList);
+    folderMailboxTree.value = await _treeBuilder.generateMailboxTree(folderMailboxList)
+        .then((tree) {
+          tree.root.childrenItems?.sort((node, other) => node.item.name?.compareAlphabetically(other.item.name) ?? -1);
```

Comment:
why  default `-1`

Reply:
if this node is `null`, it should be below `other`